From 0cf605b2529056797bbe7869ce7e961dd364a428 Mon Sep 17 00:00:00 2001
From: Tonko Mulder <tonko@tonkomulder.nl>
Date: Wed, 3 Oct 2012 16:29:53 +0200
Subject: [PATCH] adjusted for server use

---
 .tmux.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.tmux.conf b/.tmux.conf
index 09dc894..6af2f74 100644
--- a/.tmux.conf
+++ b/.tmux.conf
@@ -3,7 +3,7 @@ set-window-option -g utf8 on
 set -g prefix C-a
 unbind %
 unbind C-b
-bind C-a send-prefix
+bind C-s send-prefix
 
 # Force a reload of the config file
 unbind r
-- 
1.7.12.1

